app/src/main/java/ch/epfl/sdp/peakar/user/profile/NewProfileActivity.java

```diff
@@ -50,7 +59,7 @@
 
     private boolean isAuthProfile;
     private Account displayedAccount = null;
-
+    private String otherId = null;
```

Comment:
This attribute is redundant as the id of other users can be retrieved by doing so:
`((OtherAccount)displayedAccount).getUserID()`
Should we leave it anyway for using the ID more easily?